nix/lib/purescript.nix

```diff
@@ -1,129 +1,52 @@
 { stdenv
-, lib
-, linkFarm
-, cacert
 , nodejs
-, nodePackages
-, python2
-, git
-, fetchurl
-, yarn2nix-moretea
-, nodejs-headers
 , easyPS
+, nix-gitignore
+, buildNodeModules
 }:
 
-{ psSrc
+{
+  # path to generated purescript sources
+  psSrc
+  # path to project sources
 , src
+  # name of the project
 , name
-, yarnLock ? src + "/yarn.lock"
-, yarnNix ? src + "/yarn.nix"
-, packageJSON ? src + "/package.json"
-, additionalPurescriptSources ? [ ]
+  # additional dependencies required to build node_modules (e.g available during `npm install`)
+, additionalNpmBuildInputs ? [ ]
+  # packages as generated by psc-pacakge2nix
 , packages
+  # spago packages as generated by spago2nix
 , spagoPackages
+  # web-common project
 , webCommon
-, checkPhase ? "yarn --offline test"
-, passthru ? { }
+  # test script
+, checkPhase ? ""
```

Comment:
Dead!

Reply:
oops, i should actually inherit that - thanks for spotting this one ;)